.cloud-build/execute_changed_notebooks_helper.py

```diff
@@ -110,6 +115,31 @@ def _process_notebook(
         nbformat.write(nb, new_file)
 
 
+def _get_notebook_python_version(notebook_path: str) -> str:
+    """
+    Get the python version for running the notebook if it is specified in
+    the notebook.
+    """
+    python_version = PYTHON_VERSION
+
+    file = open(notebook_path) # Open the ipynb file that is passed in
```

Comment:
For consistency: Maybe stick with having comments on top of the line instead of to the right

Reply:
Done.